### ISSUE TITLE
This takes backups from each control plane node, which is strictly prohibited in the documentation.

### ISSUE THREAD

**Issue by kalloa** [lgchiaretto/openshift4-backup-automation#8]
I don't get it why but the documentation prohibits taking backup on all control planes, see:
<img width="561" alt="Screenshot 2022-07-15 at 10 04 34" src="https://user-images.githubusercontent.com/8790149/179180668-ffec8aad-d3b3-47fe-b352-6c5a31a711a8.png">
I'm referring to this: https://docs.openshift.com/container-platform/4.10/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html

did you try to restore from a backup already? did it work without any issue?

**Comment by lgchiaretto** [comment]
You don't need to create in all master nodes because you just need only one master to restore your cluster. I prefer to create in all of them because I have the backup file on all master nodes and no matter which node I lose I'll have the backup in another.

_did you try to restore from a backup already? did it work without any issue?_

Yes, I did. I tested many scenarios including crashing two master nodes and restoring the cluster using only one master node.
